packages/mocks/tsconfig.json

```diff
@@ -1,11 +1,10 @@
 {
   "extends": "../../config/tsconfig.base.json",
 
-  "include": ["./src/**/*"],
+  "include": ["./src/**/*", "./src/**/*.json"],
 
   "compilerOptions": {
     "outDir": "./dist",
-    "rootDir": "./src",
-    "composite": true
```

Comment:
are there any reasons to remove the `composite` flag? it seems beneficial according to docs -> https://www.typescriptlang.org/tsconfig#composite

Reply:
yes,
a. we do not use it as path in the plugin, so it's not doing anything
b. it does not play nice with rollup, specifically rollup don't know how to handle `.json` files with composite option.